dotnet/CoreLib/Handlers/TextExtractionHandler.cs

```diff
@@ -107,8 +111,8 @@ public class TextExtractionHandler : IPipelineStepHandler
                     if (string.IsNullOrWhiteSpace(url))
                     {
                         skipFile = true;
-                        uploadedFile.Log(this, "The web page URL is emtpy");
-                        this._log.LogWarning("The web page URL is emtpy");
+                        uploadedFile.Log(this, "The web page URL is empty");
+                        this._log.LogWarning("The web page URL is empty");
```

Comment:
thanks :-) oddly my IDE didn't catch this, I usually get red squiggles for typos